applications/registration-react/src/config.js

```diff
@@ -70,5 +70,7 @@ export const loadConfig = async () => {
   //   SSO_HOST: 'http://localhost:8084'
   // });
 
+  // use local server on host.
+  // env.REGISTRATION_API_HOST = 'http://localhost:8115';
```

Comment:
Can this be removed entirely?

Reply:
ok, flytter til fdk-docker-compose .envrc.template

Reply:
fikset